civitai-downloader-v2/scripts/quick_download.py

```
@@ -0,0 +1,353 @@
+        
+        if not models:
+            logger.error("❌ No models found matching criteria")
+            return
+        
+        # Sort by download count (Most Downloaded first)
+        def get_download_count(model):
+            try:
+                import ast
+                raw_data = ast.literal_eval(model[6]) if model[6] else {}
+                return raw_data.get('stats', {}).get('downloadCount', 0)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)